app/controllers/nonsipp/memberdetails/SchemeMembersListController.scala

```diff
@@ -180,12 +184,33 @@ class SchemeMembersListController @Inject()(
               }
           }.merge.pure[Future],
           value => {
-            if (lengthOfMembersDetails == maxSchemeMembers && value) {
+            val notChecked = request.userAnswers.get(MembersDetailsChecked(srn)).contains(false)
+
+            if (notChecked) {
+              for {
+                updatedAnswers <- Future.fromTry(
+                  request.userAnswers
+                    .set(MembersDetailsChecked(srn), true)
+                )
+                _ <- saveService.save(updatedAnswers)
+                _ <- psrSubmissionService
+                  .submitPsrDetailsWithUA(
+                    srn,
+                    updatedAnswers,
+                    fallbackCall = routes.SchemeMembersListController.onPageLoad(srn, page, manualOrUpload)
```

Comment:
This fallback call is where the error page should navigate when the submission (POST) fails.
Would it be better to navigate to the task list?
`fallbackCall = controllers.nonsipp.routes.TaskListController.onPageLoad(srn)`